src/docs/product/dashboards/index.mdx

```diff
@@ -16,13 +16,22 @@ If you’re interested in being an Early Adopter, you can turn your organization
 
 </Note>
 
-Dashboards provide the ability to build custom views of your application's health, including the ability to navigate issues and performance across multiple projects. Dashboards are made up of one or more widgets, and each widget visualizes one or more [Discover queries](/product/discover-queries/).
+Dashboards will enable you to get a broad overview of your application’s health. You’ll be able to navigate through error and performance data across multiple projects and zoom in on any time series visualizations you may want to investigate. All widgets in the same view will update synchronously to reflect the new date range in the Global Selection Header.
 
-![Widgets visualizing events, errors by country, affected users, and handled vs unhandled issues.](dashboard.png)
+![Widgets visualizing events, errors by country, affected users, and handled vs unhandled issues.](dashboard_default.png)
 
-You can filter the information that's displayed in a dashboard in different ways:
+## Default View
+The view you’ll start out with will contain the following widgets:
```

Comment:
I'm not sure why we wouldn't refer to this as the Default Dashboard. I don't know if it's helpful to start using the term "view" and "dashboard" interchangeably when we can just use one that refers back to the feature. I've made my suggestion below with that in mind.

```suggestion
## Default Dashboard
The default dashboard includes the following widgets:
```

Reply:
Changed